FormaticTestUtils.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { mount } from 'enzyme';
```

Comment:
Any thought to using `react-testing-library`?

Reply:
I hadn't but maybe I should. It just wasn't another rabbit hole I wanted to go down right now.